DataFormats/TrackerRecHit2D/BuildFile.xml

```diff
@@ -5,6 +5,7 @@
 <use   name="DataFormats/Phase2TrackerCluster"/>
 <use   name="FWCore/MessageLogger"/>
 <use   name="Geometry/CommonDetUnit"/>
+<use   name="Geometry/TrackerGeometryBuilder"/>
```

Comment:
I'm not sure if this dependence is allowed (DataFormats have special restrictions, this can be ok, I don't know for sure)